staging/src/k8s.io/kubectl/pkg/kuberc/marshal.go

```diff
@@ -59,6 +68,12 @@ func decodePreference(kubercFile string) (*config.Preference, error) {
 		attemptedItems++
 		pref, gvk, strictDecodeErr := scheme.StrictCodecs.UniversalDecoder().Decode(doc, nil, nil)
 		if strictDecodeErr != nil {
+			// check whether the client-go credential plugin allowlist is
+			// causing the strict parsing failure
+			if pluginPathErr, ok := asAllowlistErr(strictDecodeErr); ok {
+				return nil, fmt.Errorf("%w: %w", pluginAllowlistError, pluginPathErr)
```

Comment:
```suggestion
				return nil, pluginPathErr
```

I don't know if multiple `%w` are allowed and I think `pluginPathErr` already includes `pluginAllowlistError`?

Reply:
done